src/main/drivers/pwm_output.c

```diff
@@ -280,3 +281,28 @@ void servoInit(const servoConfig_t *servoConfig)
 }
 
 #endif
+
+#ifdef BRUSHED_ESC_AUTODETECT
+uint8_t hardwareMotorType = MOTOR_UNKNOWN;
```

Comment:
I can continoe to fix this but "_et" is not yet used at all in the BF code? Only "_e" or "_t".